18/README.md

```diff
@@ -142,6 +142,9 @@ Application developers can use the following environment variables to configure
 **`NPM_RUN`**  
        Select an alternate / custom runtime mode, defined in your `package.json` file's [`scripts`](https://docs.npmjs.com/misc/scripts) section (default: npm run "start"). These user-defined run-scripts are unavailable while `DEV_MODE` is in use.
 
+**`NODE_CMD`**  
```

Comment:
This does not sound quite right now as it's only true if INIT_WRAPPER is set to true right?